node/src/command.rs

```diff
@@ -296,10 +310,20 @@ pub fn run() -> Result<()> {
 		}
 		None => {
 			let runner = cli.create_runner(&cli.run.normalize())?;
+			let chain_spec = &runner.config().chain_spec;
+			let is_pint_dev = chain_spec.is_dev();
```

Comment:
What about 
```suggestion
                      let is_pint_dev = cli.base.base.shared_params.dev; 
```

Reply:
this will probably not work for `chain pint-dev`

Reply:
ah I think I can follow now.
yes lets make this change